eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/Types.hs

```diff
@@ -16,6 +16,7 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE TypeOperators #-}
 {-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-deprecations #-}
```

Comment:
I checked that trying to use deprecated types from this module (eg: KeyPairs), WILL result in deprecation error, so this no-deprecation applies within this file rather than affecting the usages.

Reply:
awesome, good to know, thank you!